treeherder/model/migrations/0033_add_branch_field_to_repository.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0032_bugjobmap_jobnote_models'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='repository',
+            name='branch',
+            field=models.CharField(default='master', max_length=50),
```

Comment:
`master` is only the default branch name for Git, whereas Hg uses `default`.

As such, should we just make this a nullable field, and only set it for Git repos?